src/holofuel/pages/TransactionHistory/TransactionHistory.test.js

```diff
@@ -344,7 +346,8 @@ describe('TransactionHistory', () => {
       expect(cancelPendingRequestMock.newData).toHaveBeenCalled()
     })
 
-    it('should open CancellationModal and trigger HolofuelCancelMutation for Pending Offer', async () => {
+    // NB: Intentionally skipped all Cancellation funcationality until DNA/Core cancellation bug is resolved.
```

Comment:
can you add something searchable to all these comments so that we can find them. how about 
`// CANCEL-BUG: Intentionally skipped all Cancellation funcationality until DNA/Core cancellation bug is resolved.`